Topics/Development_Process/App_Store_Deployment.md

```diff
@@ -0,0 +1,144 @@
+
+If you chose to use **Ad Hoc**, however, you would build a .ipa which you can send to your testers to install onto their device or device simulator on Xcode.
+
+For further clarification about App Distribution, you can refer to these **Apple's Documentation** links below:
+
+- [Distributing your app for beta testing and releases](https://developer.apple.com/documentation/xcode/distributing-your-app-for-beta-testing-and-releases)
+
+- [Distributing your app to registered devices](https://developer.apple.com/documentation/xcode/distributing-your-app-to-registered-devices)
+
+### App Store Submission 
+There will be a full review process, which should be completed within 24 hours according to Apple, but it may take longer depending on your app.
+In comparison to TestFlight's review process, since this would go to the official App Store, they will be enforcing stricter standards.
+
+It would be good practice to continuously develop your app keeping proper safety and privacy guidelines in mind, but in case you would like to assess on your own prior to submission, you could refer to these specific review guidelines:
+
+- [App Store Review Guidelines](https://developer.apple.com/app-store/review/guidelines/)
+
+- [Human Interface Design Guidelines](https://developer.apple.com/design/human-interface-guidelines/)
+
+- [Guidelines for Using Apple Trademarks and Copyrights](https://www.apple.com/legal/intellectual-property/guidelinesfor3rdparties.html)
+
+
+If you used [TestFlight](#testflight) for beta testing earlier, you should be familiar with this process as it is very similar for the App Store Review. If you used Ad hoc to test, then you may refer to the steps mentioned in the TestFlight process in getting the .ipa uploaded onto App Store Connect.
+
+Now you are ready to submit your app for review in App Store Connect. 
+This is what you can consider to be your dashboard in managing your apps. 
+You are able to view sales, access app analytics, as well as submit your app for App Store Review.
+
+After selecting the app to 'Add for Review' you must also provide:
+
+- Information to provide App Store Reviewers:
+    - Contact information
+    - Notes/Documentation
+    - Demo Account (if applicable)
+- Description and Features
+- Any extra information that pertains to how you would like your app to be presented
+    - App banners, app promotions, etc.
+
+Congratulations! From this point on, all you have to do is sit back and wait for the review decision to come out. Make sure to make the appropriate changes if they were requested or download your app from the official App Store if it was approved!
+
+For further information on the App Review process, please consult Apple's [App Review Docs](https://developer.apple.com/app-store/review/#:~:text=On%20average%2C%2090%25%20of%20submissions,app%20for%20iPhone%20and%20iPad.). This contains all the steps, common issues, as well as contact information if you need further help in the app review process.
```

Comment:
In alignment with the table of contents, this could be linked there as well so that readers can skip to it if they want.

Reply:
okay, I added a section and header about this